swarm.cabal

```diff
@@ -149,70 +298,73 @@ library
-                      http-client                   >= 0.7 && < 0.8,
-                      http-client-tls               >= 0.3 && < 0.4,
-                      http-types                    >= 0.12 && < 0.13,
                       lens                          >= 4.19 && < 5.3,
                       linear                        >= 1.21.6 && < 1.23,
-                      lsp                           >= 1.6 && < 1.7,
-                      megaparsec                    >= 9.0 && < 9.4,
-                      minimorph                     >= 0.3 && < 0.4,
                       mtl                           >= 2.2.2 && < 2.3,
-                      murmur3                       >= 1.0.4 && < 1.1,
                       natural-sort                  >= 0.1.2 && < 0.2,
-                      parser-combinators            >= 1.2 && < 1.4,
-                      prettyprinter                 >= 1.7.0 && < 1.8,
-                      random                        >= 1.2.0 && < 1.3,
-                      servant                       >= 0.19 && < 0.20,
-                      servant-server                >= 0.19 && < 0.20,
-                      SHA                           >= 1.6.4 && < 1.6.5,
-                      simple-enumeration            >= 0.2 && < 0.3,
                       split                         >= 0.2.3 && < 0.3,
-                      stm                           >= 2.5.0 && < 2.6,
-                      syb                           >= 0.7 && < 0.8,
-                      tagged                        >= 0.8 && < 0.9,
-                      template-haskell              >= 2.16 && < 2.20,
                       text                          >= 1.2.4 && < 2.1,
-                      text-rope                     >= 0.2 && < 0.3,
-                      text-zipper                   >= 0.10 && < 0.14,
+                      text-zipper                   >= 0.10 && < 0.13,
                       time                          >= 1.9 && < 1.14,
-                      unification-fd                >= 0.11  && < 0.12,
-                      unordered-containers          >= 0.2.14 && < 0.3,
                       vector                        >= 0.12 && < 0.14,
-                      vty                           >= 5.33 && < 5.39,
+                      vty                           >= 5.33 && < 5.38,
+                      witch                         >= 1.1.1.0 && < 1.2,
+                      word-wrap                     >= 0.5 && < 0.6,
+                      warp                          >= 3.2 && < 3.4,
+                      swarm-util,
+                      swarm-language,
+                      swarm-game,
+                      swarm-version,
+    hs-source-dirs:   src/swarm-user-interface
+    default-language: Haskell2010
+    default-extensions:
+      -- Avoid unexpected unevaluated thunk buildup
+      -- See discussion in #415
+      StrictData
+
+
+library
```

Comment:
Why doesn't this `library` have a name?

Reply:
It did not before, it is simply `swarm`. I am not 100% clear about whether we need this or if `swarm-app` would be better or maybe if it could even merge with `swarm-exe`.

I do not think we need to do anything about this right now.